docs/source/technical_tutorials/access/accessing_external_data.ipynb

```diff
@@ -76,7 +76,7 @@
   },
```

Comment:
<p>Also, mention link to <a href="https://maap-project.org/profile" target="_blank">https://maap-project.org/profile/</a> for getting the PGT token.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>